storage/mysql/log_storage_test.go

```diff
@@ -1117,6 +1125,15 @@ func ensureAllLeavesDistinct(leaves []*trillian.LogLeaf, t *testing.T) {
 	}
 }
 
```

Comment:
I see only changes to Dequeue-based tests. We should test that we're getting the correct timestamps from GetLeaves* methods too.

Reply:
Done.